docs/install/containerd/containerd-install-guide.md

```diff
@@ -0,0 +1,99 @@
+# Install Kata Containers with containerd
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && { echo >&2 "ERROR: Please remove existing Kata Containers installation"; exit 1; }
+   ```
+
+1. Install dependencies
+
+   ```bash
+   $ source /etc/os-release || source /usr/lib/os-release
+   $ packages='containerd curl git jq'
+   $ case "$ID" in
+   $   debian|ubuntu) sudo apt-get -y install $packages ;;
+   $   *) echo >&2 "ERROR: Unsupported distro: $ID"; exit 1 ;;
+   $ esac
+   ```
+
+1. Install Kata Containers
+
+   ```bash
+   $ tmpdir=$(mktemp -d)
+   $ pushd "$tmpdir" &>/dev/null
+   $ url="https://api.github.com/repos/kata-containers/runtime/releases/latest"
```

Comment:
Shouldn't repo name be `kata-containers/kata-containers` ?

Reply:
Not here - if you look carefully, this is the API URL rather than the "human readable" URL. This one returns JSON rather than XHTML ;)

Reply:
Oh - sorry, I misunderstood what you were referring to. Yes we currently produce releases at https://github.com/kata-containers/runtime/releases, but I think we need to start using https://github.com/kata-containers/kata-containers/releases for 2.x releases. I've raised https://github.com/kata-containers/kata-containers/issues/745 to discuss this.

Note that the decision will affect https://github.com/kata-containers/kata-containers/pull/735, which needs to know where to look for releases.